Resources/config/services.xml

```diff
@@ -17,12 +17,14 @@
     </parameters>
 
     <services>
-        <service id="bg_object_routing.adapter.symfony_21" class="%bg_object_routing.adapter.symfony_21.class%">
+        <service id="bg_object_routing.adapter.symfony_22" class="JMS\ObjectRouting\Symfony\Symfony22Adapter" public="false">
```

Comment:
@mpdude can you use a parameter instead to stay consistent with the rest of the code? You can replace ```bg_object_routing.adapter.symfony_21.class``` by yours.

Thanks